test/membrane_rtc_engine/tee_test.exs

```diff
@@ -303,14 +302,34 @@ defmodule Membrane.RTC.Engine.TeeTest do
     assert_sink_event(pipeline, :sink, ^event)
   end
 
+  test "VoiceActivityChanged on video causes raise" do
+    track = build_h264_track()
+
+    assert_raise RuntimeError, fn ->
+      Membrane.RTC.Engine.Tee.handle_event(
+        Membrane.Pad.ref(:input, {0, :low}),
```

Comment:
```suggestion
        Membrane.Pad.ref(:input, {0, :high}),
```

Reply:
It literally doesn't matter. It's a **video** track

Reply:
True that, I was sure it's an audio one :man_facepalming: